src/nvapi_gpu.cpp

```diff
@@ -3,6 +3,10 @@
 #include "util/util_statuscode.h"
 #include "util/util_string.h"
 
+#if defined(__GNUC__)
+#define _ReturnAddress() __builtin_return_address(0);
```

Comment:
I think we can move this define to `nvapi_private.h`.

Reply:
Sure